src/index.html

```diff
@@ -3,144 +3,144 @@
     />
+    <link rel="stylesheet" href="./css/style.scss" />
     <script
       type="text/javascript"
       src="https://npmcdn.com/parse/dist/parse.min.js"
     ></script>
-    <!-- <link rel="stylesheet" href="./css/popups.css" /> -->
+    <script src="https://unpkg.com/magic-grid/dist/magic-grid.cjs.js"></script>
+
     <title>places-been</title>
   </head>
   <body>
     <!-- N A V B A R  -->
     <nav class="navbar">
-      <div class="container container--space-between">
-        <a href="#" class="navbar-logo">
-          <h1 class="navbar__title">places been</h1>
-          <h2 class="navbar__subtitle">a travel journal</h2>
-        </a>
-        <ul class="navbar__links">
-          <li class="navbar__link">
-            <a href="#">About</a>
-          </li>
-          <li class="navbar__link">
-            <a href="#main-buttons">My cities</a>
-          </li>
-          <li class="navbar__link">
-            <a href="#">Contact</a>
-          </li>
-        </ul>
-      </div>
-      <!-- <ul class="navbar--social container--social">
-        <li class="social__links">
-          <a href="#">Ig</a>
+      <ul class="navbar__links">
+        <li class="navbar__list">
+          <a class="navbar__link" href="#">Hey,</a>
         </li>
-        <li class="social__links">
-          <a href="#">Fb</a>
+        <li class="navbar__list">
+          <a class="navbar__link" href="">Stories,</a>
         </li>
-      </ul> -->
+        <li class="navbar__list">
+          <a class="navbar__link" href="#">About</a>
+        </li>
+      </ul>
     </nav>
-    <!-- H E A D E R -->
-    <header class="header container--main">
-      <img class="header__img" src="./img/header2.jpg" alt="" />
+
+    <!-- H E A D E R  -->
+    <header class="header">
+      <!-- slideshow -->
+      <section class="container--slideshow">
+        <div class="slides">
+          <!-- <div class="slide slide-1">
+            <img src="./img/photo3.jpg" alt="" />
+          </div> -->
+          <div class="slide slide-1">
+            <img src="./img/photo1.jpg" alt="" />
+          </div>
+          <div class="slide slide-2">
+            <img src="./img/photo8.jpg" alt="" />
+          </div>
+          <div class="slide slide-3">
+            <img src="./img/photo25.jpg" alt="" />
+          </div>
+          <div class="slide slide-4">
+            <img src="./img/photo0.jpg" alt="" />
+          </div>
+        </div>
+        <div class="slide-controls">
+          <button class="prev-btn"></button>
+          <button class="next-btn"></button>
+        </div>
+      </section>
+      <!-- headline -->
+      <section class="heading container--heading">
+        <h2 class="heading__subtitle">Hey, welcome and take a look</h2>
+        <h1 class="heading__title">
+          This is a photographic journal of
+          <span class="heading__name">Maja Cvetkovic</span>, an amateur
+          photographer based in Nis, Serbia.
+        </h1>
```

Comment:
Imagine this website without styles. like a raw book. Does it make sense for these two headings to be h2 and h1?

The first heading in a page must be always a `<h1>`.

<details>
  <summary>My proposal for solution:</summary>

Hide visually the page title to keep semantics using [visually-hidden](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/). The other two texts are just raw text.
```html
<h1 class="visually-hidden">Maja Cvetkovic - Photographic Journal</h1>
<p>Hey, welcome...</p>
<p>This is a phtog...</p>
```

</details>

Reply:
It hasn't even crossed my mind, I'll try to pay more attention to that